scripts/protocgen.sh

```diff
@@ -11,7 +11,7 @@ cd proto
 proto_dirs=$(find ./cosmos -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
 for dir in $proto_dirs; do
   for file in $(find "${dir}" -maxdepth 1 -name '*.proto'); do
-    if grep "option go_package" $file &> /dev/null ; then
+    if grep "option go_package.*(?!cosmos-sdk/api)" $file &> /dev/null ; then
```

Comment:
this is to skip gogo generation for files which are natively generated with pulsar

Reply:
nit: I would add a comment above the statement describing what is going on.